packaging/post_build_script_linux.sh

```diff
@@ -7,8 +7,8 @@ for pkg in dist/torchdata*.whl; do
     echo "PkgInfo of $pkg:"
     pkginfo $pkg
 
-    auditwheel repair $pkg --plat manylinux2014_x86_64 -w wheelhouse
```

Comment:
Looks like we are mixing manylinux here and conda removal

Reply:
Thanks, updated (reverted).